lib/pure/math.nim

```diff
@@ -90,7 +90,7 @@ proc isPowerOfTwo*(x: int): bool {.noSideEffect.} =
 proc nextPowerOfTwo*(x: int): int {.noSideEffect.} =
   ## returns `x` rounded up to the nearest power of two.
   ## Zero and negative numbers get rounded up to 1.
-  result = x - 1 
+  result = x + ord(x<0) - 1
```

Comment:
Meh, who cares about nextPowerOfTwo for negative numbers? What's the use case? Alternatively 'nextPowerOfTwo' should take 'x: Natural'.